front/cypress/integration/admin/idea_manager.ts

```diff
@@ -132,7 +132,10 @@ describe.skip('Idea manager', () => {
     });
   });
 
-  describe('Assignee select', () => {
+  // I don't know how this is supposed to work, but I think we should
+  // just create a new idea and make it unassigned before we do this check,
+  // because it's now failing because there are no unassigned ideas.
```

Comment:
Where does it fail? It seems CI is green now.

Reply:
I mean in `dev-notifications-e2e-tests` channel

Reply:
The entire suite is being skipped right now (see line 3)

Reply:
Ah indeed. Sorry for the noise then.